src/cmd/cmd.zig

```diff
@@ -82,18 +83,30 @@ pub fn run() !void {
 // *                       CLI COMMANDS                       *
 // ************************************************************
 
+/// Errors that occur because the user misused the CLI.
+const UsageError = error{
+    /// Occurs when the user requests a config that is not supported by the current build.
+    ///
+    /// For example because execution traces are globally disabled.
+    incompatibleBuildOptions,
```

Comment:
please start with uppercase

Reply:
Didn't even notice it lol. Way to used to writing everything in lowercase in C.